robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -216,6 +216,11 @@ def fill_template_form(self,**kwargs):
                         self.selenium.wait_until_page_does_not_contain_element(locator,error="could not open datepicker")
                     else:
                         self.salesforce._populate_field(field_loc,value) 
+                elif section=="Field Label":
```

Comment:
Where are you checking for the section value ?
I see the usage in your test as this:
```
Fill Template Form                      
    ...                                     Opportunity: Close Date=&{DATE}
    ...                                     Opportunity: Campaign ID=&{CAMPAIGN}
    ...                                     Payment: Check/Reference Number=&{PAY}
    ...                                     Opportunity: Record Type ID=&{RECORD_TYPE} 
```

I feel like I am missing something obvious. Can u explain once ?

Reply:
here each variable is a dictionary that contains key as Field Label and Value as some value Ex: Field Label=Closed Date